### ISSUE TITLE
Dns spoofing causes the internet to become unresponsive 

### ISSUE THREAD

**Issue by asciiterminal** [DanMcInerney/dnsspoof#3]
python dnsspoof.py -r <router's ip> -v <victim's ip> -d <facebook.com>

after running this i go on to my phone and type in to the browser facebook.com and the internet becomes unresponsive

**Comment by DanMcInerney** [comment]
Yeah this tool was just practice for implementing it in LANs.py I don't
know that I ever killed all the bugs in it. But I don't dev either this
tool or LANs.py anymore. You're better off using bettercap.
